splinepy/microstructure/tiles/double_lattice.py

```diff
@@ -57,19 +57,19 @@ def create_tile(
         # set to default if nothing is given
         if parameters is None:
             self._logd("Tile request is not parametrized, setting default 0.2")
-            parameters = _np.ones((1, 2)) * 0.1
-        # Maintain backwards compatibility
-        elif parameters.shape[1] == 1:
-            _warning("DoubleLattice now expects 2 values")
-            index_second_value = 0
-            self._n_info_per_eval_point = 1
+            parameters = _np.ones((1, self._n_info_per_eval_point)) * 0.1
```

Comment:
```suggestion
            parameters = _np.ones((1, self._n_info_per_eval_point)) * 0.1
                    # Maintain backwards compatibility
         elif parameters.shape[1] == 1:
             _warning("DoubleLattice now expects 2 values")
             self._index_second_value = 0
             self._n_info_per_eval_point = 1
```

It's cleaner if the index is a member variable. Another option is to discard the variable completely and replace the calls of index using `n_info-1`, but that is very hard to read. You could also throw a deprecation warning.

Reply:
Member variable definitely makes sense. I also thought about the n_info - 1 and I also think it makes it harder to read.